src/test/java/guitests/UITest.java

```diff
@@ -362,6 +362,29 @@ private void awaitCondition(Callable<Boolean> condition, int timeoutInSeconds) {
     }
 
     /**
+     * Performs logout from File -> Logout on HubTurbo's pView.
+     */
+    public void logout(){
+        clickMenu("File", "Logout");
+    }
+
+    /**
+     * Performs UI login on the login dialog box. Assumes the cursor is on the login dialog box but nothing selected.
```

Comment:
Nope it's not necessary. If nothing is selected, `selectAll` will select the whole filter text anyways.